crates/gpui/src/platform/keystroke.rs

```diff
@@ -109,6 +109,15 @@ impl Keystroke {
             }
         }
 
+        #[cfg(target_os = "macos")]
```

Comment:
I don't think we want to remove the ability to respond to `Fn` keys at all, both Zed and Chrome use a function key binding to enable full screen on macOS.